### ISSUE TITLE
Some users cannot resolve api.faforever.com

### ISSUE THREAD

**Issue by Sheikah45** [FAForever/downlords-faf-client#2374]
**Describe the bug**
From the forum:
updated to 1.6 today, after having not played for a good while, however when i try to log in i get the following message

java.util.concurrent.CompletionException: org.springframework.web.reactive.function.client.WebClientRequestException: failed to resolve ' api.faforever.com ' after 2 queries ; nested exception is java.net.UnknownHostException: failed to resolve ' api.faforever.com ' after 2 queries 

**To Reproduce**
Some minority of users cannot resolve api.faforever.com using the webclient from spring

**Expected behavior**
Users should be able to easily connect to the api like they have with client 1.5.0

**Wanna have the bug fixed quickly?**
Visit Issue hunt...
[![Issue hunt](https://github.com/BoostIO/issuehunt-materials/raw/master/v1/issuehunt-button-v1.svg?sanitize=true)](https://issuehunt.io/r/FAForever/downlords-faf-client)

**Comment by micheljung** [comment]
Failing to resolve `api.faforever.com` is a system error and can basically only happen if they are offline or have a bad DNS configured (given that FAF or their DNS provider didn't mess things up). Either way, there is nothing the client can do other than maybe displaying a human-readable message that says "You're either offline or someone messed something up and you can't do anything about it" 😄 

But it's very unlikely to be a bug in the client unless there are actually spaces around ` api.faforever.com `.

**Comment by bukajsytlos** [comment]
Yeah it makes not much sense, but apparently there is no issue with 1.5.
But resolve timeout could be issue although I would assume different
exception

**Comment by Sheikah45** [comment]
Yes I suspect that issue is either potentially a bug in how the NettyDnsResolver is resolving the name or the timeout value changed. As there were no users that had this issue in 1.5.0 when we used the restTemplate. 
 With the restTemplate we also used the apache http client which has its own resolver. So another soultion may be to use their async http client with the webclient. For now I tried just increasing the default timeout as that was the most obviouserror in the stack traces I had seen.

But this is definitely caused by either a change in a library or our code as it was not an issue previously.
